web/src/main/ui/app/components/flows-new/edit-flow/edit-flow.component.ts

```diff
@@ -180,7 +174,6 @@ export class EditFlowComponent implements OnInit, OnDestroy {
   }
   updateStep(step) {
     this.manageFlowsService.updateStep(this.flow.id, step.id, step).subscribe(resp => {
-      this.editFlowUi.stepUpdated(step);
```

Comment:
It looks like some functionality that's on develop has been removed with this PR. Please look into this